TypeCobol.LanguageServer/TypeCobolCustomLanguageServer/Document Configuration Changed Notification/DocumentConfigurationChangedNotification.cs

```diff
@@ -0,0 +1,9 @@
+﻿using TypeCobol.LanguageServer.JsonRPC;
+
+namespace TypeCobol.LanguageServer.TypeCobolCustomLanguageServerProtocol
+{
+    class DocumentConfigurationChangedNotification
+    {
+        public static readonly NotificationType Type = new NotificationType("typecobol/documentConfigurationChanged", typeof(DocumentConfigurationChangedParams));
```

Comment:
This notification has 2 purposes:
- Associate a textDocument with a project
- Update copy folders associated with a project
Is that correct?

Reply:
Yes because, from the client point of view both points are related.